src/add-remove.test.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { AssignButtons, AddTask, DeleteTask } from './add-remove.js';
+
+document.body.innerHTML = `<div class="list-container">
+  <div class="list-title">
+    <h4>Todays To Do</h4>
+    <button type="button">
+      <img src="https://media.flaticon.com/dist/min/img/uicons/icons/rounded/regular/fi-rr-rotate-right.svg" width="14" alt="Refresh">
+    </button>
+  </div>
+  <div class="list-add">
+    <input type="text" placeholder="Add to your list..." id="input-add">
+    <button type="button" id="button-add">
+      <img src="https://media.flaticon.com/dist/min/img/uicons/icons/rounded/regular/fi-rr-add.svg" width="14" alt="Add task">
+    </button>
+  </div>
+  <div class="list-content"></div>
+  <button type="button" id="button-clear">Clear all Completed</button>
+</div>;`;
+
+const fakeLocalStorage = (() => {
+  let store = {};
+
+  return {
+    getItem(key) {
+      return store[key] || null;
+    },
+    setItem(key, value) {
+      store[key] = value.toString();
+    },
+    removeItem(key) {
+      delete store[key];
+    },
+    clear() {
```

Comment:
[OPTIONAL]
- I would recommend to use separate file to use mock for funtions that used `localStorage`, `DOM` or `external APIs`.
- Also, I would recommend to put all the test file in a separate test folder so that the structure of your project look good.